sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/certificate/aio/_configuration.py

```diff
@@ -8,36 +8,35 @@
 
 from typing import Any
 
-from azure.core.configuration import Configuration
 from azure.core.pipeline import policies
 
-from ..._version import VERSION
+from .._version import VERSION
 
 
-class ConfidentialLedgerCertificateClientConfiguration(Configuration):  # pylint: disable=too-many-instance-attributes,name-too-long
+class ConfidentialLedgerCertificateClientConfiguration:  # pylint: disable=too-many-instance-attributes,name-too-long
     """Configuration for ConfidentialLedgerCertificateClient.
 
     Note that all parameters used to create this instance are saved as instance
     attributes.
 
-    :param certificate_endpoint: The certificate endpoint (or "Identity Service Endpoint" in the
-     Azure portal), for example https://identity.confidential-ledger.core.azure.com. Required.
-    :type certificate_endpoint: str
-    :keyword api_version: Api Version. Default value is "2022-05-13". Note that overriding this
-     default value may result in unsupported behavior.
+    :param endpoint: The certificate endpoint (or "Identity Service Endpoint" in the Azure portal),
+     for example https://identity.confidential-ledger.core.azure.com. Required.
+    :type endpoint: str
+    :keyword api_version: Api Version. Default value is "2024-08-22-preview". Note that overriding
+     this default value may result in unsupported behavior.
     :paramtype api_version: str
     """
 
-    def __init__(self, certificate_endpoint: str, **kwargs: Any) -> None:
-        super(ConfidentialLedgerCertificateClientConfiguration, self).__init__(**kwargs)
-        api_version = kwargs.pop("api_version", "2022-05-13")  # type: str
+    def __init__(self, endpoint: str, **kwargs: Any) -> None:
+        api_version: str = kwargs.pop("api_version", "2024-08-22-preview")
```

Comment:
It looks like endpoint was renamed to certificate_endpoint previously. I'm assuming this change was made manually since the generator doesn't know about it...